.github/workflows/ci.yml

```
@@ -0,0 +1,74 @@
+name: ci
+
+"on":
+  push:
+    paths:
+      - ".github/workflows/ci.yml"
+      - "lib/**"
+      - "*.gemspec"
+      - "spec/**"
+      - "Rakefile"
+      - "Gemfile"
+      - ".rubocop.yml"
+  pull_request:
+    branches:
+      - master
+  create:
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby:
+          - "2.7"
+        db:
+          - sqlite3
+          - mysql
+          - postgresql
+    env:
+      DB: ${{matrix.db}}
+    steps:
+      - uses: actions/checkout@v1
+      - name: Install package dependencies
+        run: "[ -e $APT_DEPS ] || sudo apt-get install -y --no-install-recommends $APT_DEPS"
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{matrix.ruby}}
+      - name: Install latest bundler
+        run: |
+          gem install bundler --no-document
+      - name: Bundle install
+        run: bundle install --jobs 4 --retry 3
```

```suggestion
      - name: Set up Ruby
        uses: ruby/setup-ruby@v1
        with:
          bundler-cache: true
          ruby-version: ${{matrix.ruby}}
```

This does all the bundling for us now, and caches it

---

This one works really well ^